src/python/grpcio_test/setup.py

```diff
@@ -43,13 +52,25 @@
         'credentials/server1.pem',]
 }
 
-_INSTALL_REQUIRES = ['oauth2client>=1.4.7', 'grpcio>=0.10.0a0']
+_SETUP_REQUIRES = (
+    'pytest>=2.6',
+    'pytest-cov>=2.0',
+    'pytest-xdist>=1.11',
+)
+
+_INSTALL_REQUIRES = (
+    'oauth2client>=1.4.7', 'grpcio>=0.10.0a0'
+)
 
 setuptools.setup(
     name='grpcio_test',
-    version='0.0.1',
+    version='0.10.0a0',
     packages=_PACKAGES,
     package_dir=_PACKAGE_DIRECTORIES,
     package_data=_PACKAGE_DATA,
-    install_requires=_INSTALL_REQUIRES
+    install_requires=_INSTALL_REQUIRES + _SETUP_REQUIRES,
+    setup_requires=_SETUP_REQUIRES,
+    cmdclass={
+        'test': commands.RunTests
```

Comment:
Comma at end of line? Perhaps a module-scope constant?

(Yes, this is me being fussy about little things.)